mgmtsystem/views/res_config.xml

```diff
@@ -0,0 +1,266 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_mgmtsystem_configuration" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.mgmtsystem</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="base.res_config_settings_view_form"/>
+        <field name="priority" eval="95"/>
+        <field name="arch" type="xml">
+
+            <xpath expr="//div[hasclass('settings')]" position="inside">
+                <div class="app_settings_block" 
+                     data-string="Management System" string="Management System"
+                     data-key="mgmtsystem"
+                     groups="mgmtsystem.group_mgmtsystem_user_manager">
```

Comment:
```suggestion
                     groups="group_mgmtsystem_user_manager">
```

Reply:
this is wrong, in views you have to use the xid